pkg/controller/grafanadashboard/dashboard_controller.go

```diff
@@ -113,7 +113,7 @@ func (r *ReconcileGrafanaDashboard) Reconcile(request reconcile.Request) (reconc
 
 	// If Grafana is not running there is no need to continue
 	if !r.state.GrafanaReady {
-		log.Info("no grafana instance available")
+		log.Error(fmt.Errorf("no grafana instance available"), "")
```

Comment:
this does not have to be an error, this is expected when the operator runs without a CR created